shared/logic/Update.ts

```diff
@@ -269,6 +271,15 @@ function tickTile(xy: Tile, gs: GameState, offline: boolean): void {
    const inputWorkerCapacity = totalMultiplierFor(xy, "worker", 1, gs);
    const { total, used } = getStorageFor(xy, gs);
 
+   const options = getGameOptions();
```

Comment:
This whole block of code probably should be wrapped in an `if` that checks whether autopilot is unlocked? It's no harm to call this even if autopilot is not unlocked but this could save some performance